theme/template-parts/layout/footer-content.php

```diff
@@ -36,7 +36,8 @@ class="relative mx-auto flex max-w-7xl items-center justify-around md:px-8 borde
                                 array(
                                     'theme_location' => 'menu-2',
                                     'menu_id'        => 'footer-menu-col-1',
-                                    'items_wrap'     => '<div class="menu-name font-bold">' . wp_kses_post( $menu->name ) . '</div><ul id="%1$s" class="%2$s flex justify-start flex-col" aria-label="submenu">%3$s</ul>',
+                                    //'items_wrap'     => '<div class="menu-name font-bold">' . wp_kses_post( $menu->name ) . '</div><ul id="%1$s" class="%2$s flex justify-start flex-col" aria-label="submenu">%3$s</ul>',
```

Comment:
@amanda-gerold is there a need to keep all the commented out code? Or can we remove it to keep it cleaner?